docs/DeveloperGuide.adoc

```diff
@@ -314,16 +370,144 @@ _{More to be added}_
 == Use Cases
 
 (For all use cases below, the *System* is the `AddressBook` and the *Actor* is the `user`, unless specified otherwise)
+[discrete]
+=== Use case: Add person
+Guarantees: +
+
+1. Person is added even if input fields might be invalid (see 1a).
+
+*MSS*
+
+1.  User requests to add a person.
+2.  AddressBook adds the person.
++
+Use case ends.
+
+*Extensions*
+
+[none]
+* 1a. Either of the given NRIC, contact number, or email address is invalid.
++
+[none]
+** 1a1. AddressBook adds the person into address book.
+** 1a1. AddressBook shows a warning.
+
+* 1b. Duplicate profile is added.
++
+[none]
+** 1b1. AddressBook shows an error message and will attempt to merge the profile.
++
+
 
 [discrete]
-=== Use case: Delete person
+=== Use case: Edit person
+Guarantees: +
+
+1. Person is editted even if input fields might be invalid (see 3a).
+
+*MSS*
+
+1.  User requests to find a person
+2.  AddressBook shows details of the person
```

Comment:
Can consider changing "AddressBook" to the name of your product in your next PR :)